actions/ci-test-ts/action.yml

```diff
@@ -66,15 +66,14 @@ runs:
         run-install: "true"
       continue-on-error: true
 
-    - name: Run lint
-      if: inputs.check-only-affected != 'true'
+    - name: Run test
       shell: bash
-      run: pnpm run lint
+      run: pnpm nx run-many --targets=test --verbose
 
     - name: Run lint affected
       if: inputs.check-only-affected == 'true'
       shell: bash
-      run: pnpm run lint:affected --base=${{ inputs.base-ref }}
+      run: pnpm nx affected --targets=test --verbose
```

Comment:
I think we can remove this input now if it's not being used

```
  base-ref:
    description: ""
    required: false
    default: main
```

Reply:
Removed it, thanks